src/main/java/sc/fiji/bdvpg/bdv/navigate/ViewerTransformSyncStopper.java

```diff
@@ -0,0 +1,27 @@
+package sc.fiji.bdvpg.bdv.navigate;
+
+import bdv.util.BdvHandle;
+import net.imglib2.realtransform.AffineTransform3D;
+import net.imglib2.ui.TransformListener;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ViewerTransformSyncStopper implements Runnable {
+
+    Map<BdvHandle, TransformListener<AffineTransform3D>> bdvToTransformListener;
+
+    public ViewerTransformSyncStopper(Map<BdvHandle, TransformListener<AffineTransform3D>> bdvToTransformListener) {
```

Comment:
We should consistently use `bdv` or `bdvHandle` as object name.
(in`bdvToTransformListener`)

Reply:
oki, I changed the names in the sync actions at least, I didn't look through everything but we can do it when we find some variables named like that